django-stubs/utils/termcolors.pyi

```diff
@@ -1,14 +1,14 @@
-from collections.abc import Callable, Mapping, Sequence
-from typing import Any
+from collections.abc import Callable, Sequence
+from typing import Any, Literal
 
-color_names: Sequence
-foreground: Mapping[str, str]
-background: Mapping[str, str]
-RESET: str
-opt_dict: Mapping[str, str]
+color_names: tuple[str]
```

Comment:
Don't forget, if it's a tuple with multiple elements of the same type, you need `...`. This should fix the allowlist entry too.

```suggestion
color_names: tuple[str, ...]
```